### ISSUE TITLE
error adding image to CVE provider

### ISSUE THREAD

**Issue by rentallect** [jenkins-x/jx#867]
I ran a:

`
jx create addon anchore
`

to get a Anchore engine service running. I then ran a build, and it hit this error:

```
+ jx step post build --image 100.69.128.85:5000/foo/bar:0.0.0-SNAPSHOT-PR-2-6
adding image 100.69.128.85:5000/foo/bar:0.0.0-SNAPSHOT-PR-2-6 to CVE provider
error: error adding image to CVE provider: failed to add image 100.69.128.85:5000/foo/bar:0.0.0-SNAPSHOT-PR-2-6 to anchore engine: exit status 1
```

so I'm wondering if there is some other config I need, or if this is a jx glitch.

I'm currently running this version of JX:

```
NAME               VERSION
jx                 1.2.73
jenkins x platform 0.0.1038
kubernetes cluster v1.9.3
kubectl            v1.8.6
helm client        v2.9.0+gf6025bb
helm server        v2.9.0+gf6025bb
git                git version 2.17.0

```

Loving JX so far, btw :)

**Comment by rawlingsj** [comment]
I wonder if you try another build if it will work?  I noticed that the Anchore readiness checks pass before the application is ready to accept images.

**Comment by rentallect** [comment]
Trying another build didn't work (that was first thing I tried).  I even did a:

`jx delete addon anchore
`

followed by a second:

`jx create addon anchore
`

just to see if the initial 'create' went sour... but nothing has changed, and builds still fail.  Also, all pods in the anchore namespace in K8S look fine.

What is the best/cleanest way to back out of the CVE stuff (for now)?  I ask because when I attempt to 'delete' the addon, it says:

`release "anchore" deleted
`

which seems odd to me, since it's an addon, not a release.

Also, is it at all possible that whatever is going on here is related to the issue I experienced over in #850  ?
